cypress/integration/admin/manage-users.spec.js

```diff
@@ -0,0 +1,21 @@
+'use strict';
+
+const { loginAdmin } = require('../../actions/login');
+
+describe('Manage users', () => {
+  beforeEach(() => {
+    loginAdmin();
+  });
+
+  it('should add user', function () {
+    cy.visit('/');
+    cy.findByText(/Add User/i).click();
+    cy.findByText(/Create user/i).should('exist');
```

Comment:
Is this necessary, since the rest of the test should be sufficient to test this flow?
I'd suggest that our tests try to follow the structure:
- setup
- perform action
- assert outcome